external-crates/move/crates/move-compiler/tests/move_2024/ide_mode/colon_colon_incomplete.ide.exp

```diff
@@ -0,0 +1,95 @@
+note[I15006]: IDE path autocomplete
+   ┌─ tests/move_2024/ide_mode/colon_colon_incomplete.move:1:1
+   │  
+ 1 │ ╭ module a::m {
+ 2 │ │ 
+ 3 │ │     public enum SomeEnum {
+ 4 │ │         SomeVariant,
+   · │
+24 │ │ 
+25 │ │ }
+   │ ╰─^ Possible in-scope names: 'Option -> std::option::Option', 'SomeEnum -> a::m::SomeEnum', 'Self -> a::m', 'option -> std::option', 'vector -> std::vector', 'A -> 0x41', 'B -> 0x42', 'K -> 0x19', 'M -> 0x40', 'a -> 0x44', 'b -> 0x45', 'k -> 0x19', 'std -> 0x1', or 'sui -> 0x2'
+
+note[I15006]: IDE path autocomplete
+   ┌─ tests/move_2024/ide_mode/colon_colon_incomplete.move:10:9
+   │
+10 │         A
+   │         ^ Possible in-scope names: 'Option -> std::option::Option', 'SomeEnum -> a::m::SomeEnum', 'foo -> a::m::foo', 'member_complete -> a::m::member_complete', 'mod_complete -> a::m::mod_complete', 'pkg_complete -> a::m::pkg_complete', or 'variant_incomplete -> a::m::variant_incomplete'
```

Comment:
It's interesting that this is recognized as an access chain (to have completion info - possibly because it starts with a capital letter as using `a` will not work) but it does not include any "address" or "module" completion information but only the "member" one...

Reply:
See #18733